File: consumer-java/src/main/java/consumer/model/LeadStatus.java
package consumer.model;

import lombok.*;

@Getter @Setter
public class LeadStatus {

    private String referralId;
    private String leadId;
    private String dealId;
    private String status;

    @Override
    public String toString() {
        return "LeadStatus{" +
                "referralId='" + referralId + '\'' +
                ", leadId='" + leadId + '\'' +
                ", dealId='" + dealId + '\'' +
                ", status='" + status + '\'' +
                '}';
    }
}

File: consumer-python/model/LeadStatus.py
class LeadStatus:
    def __init__(self):
        pass

    referralId = ""
    leadId = ""
    dealId = ""
    status = ""

    def __str__(self):
        return "LeadStatus{" \
               "referralId='" + self.referralId + \
               ", leadId='" + self.leadId + \
               ", dealId='" + self.dealId + \
               ", status='" + self.status + \
               '}'